packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -32,7 +32,7 @@ The following table describes the rate limit information that is returned in the
 | ---------------------------------- | ------ | ----------------------------------------------------------------------------------------------------------------- |
 | `operationRateLimitScopeType`      | String | The type of rate limit scope affected. Example scopes: `org` or `user`                                            |
 | `operationRateLimitSecondsToReset` | String | The number of seconds that remain until the current rate limiting period ends                                     |
-| `operationRateLimitSubtype`        | String | The subtype of the rate limit event affected. Example subtypes: `email`, `phone`, `rsa_token`                     |
+| `operationRateLimitSubtype`        | String | The [subtype](#operation-rate-limit-subtypes) of the rate limit event affected. Example subtypes: `email`, `phone`, `rsa_token`|
```

Comment:
Looking at the code, it is `SMS` and `Voice call` instead of `phone`.  Probably `Email` instead of `email`.  Still hunting down `rsa_token` (it is another team).

I'll update this comment with the final answers.

Reply:
`rsa_token` is RSA SecurID, and the #eng-hybridaccess  team.  They say it doesn't use operational rate limits, and we should just drop it.